sqlalchemy_utils/view.py

```diff
@@ -6,21 +6,149 @@
 
 
 class CreateView(DDLElement):
-    def __init__(self, name, selectable, materialized=False):
+    def __init__(
+        self,
+        name,
+        selectable,
+        materialized=False,
+        if_not_exists=False,
+        or_replace=False,
+    ):
         self.name = name
         self.selectable = selectable
         self.materialized = materialized
+        self.if_not_exists = if_not_exists
+        self.or_replace = or_replace
 
 
 @compiler.compiles(CreateView)
 def compile_create_materialized_view(element, compiler, **kw):
+    return 'CREATE {}{}VIEW {}{} AS {}'.format(
+        'OR REPLACE ' if element.or_replace else '',
+        'MATERIALIZED ' if element.materialized else '',
+        'IF NOT EXISTS ' if element.if_not_exists else '',
+        compiler.dialect.identifier_preparer.quote(element.name),
+        compiler.sql_compiler.process(element.selectable, literal_binds=True),
+    )
+
+
+@compiler.compiles(CreateView, 'postgresql')
+def compile_create_materialized_view_(element, compiler, **kw):
```

Comment:
Just name all the dialect specific functions with the dialect prefix. That makes them more readable. So this function would then be `compile_create_materialized_view_postgresql`

Reply:
ok done.